tests/test_basic_flow.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+import logging
+import json
+import sys
+from os.path import expanduser
+from unittest.mock import patch
+import subprocess
+import shutil
+
+
+sys.path.append(".")
+from benchmarking.run_bench import RunBench
+
+class BasicFlowTest(unittest.TestCase):
+    test_framework = "test/test_framework"
+    test_remote_reporter = "test_remote_reporter"
+    test_remote_access_token = "test_token"
+    test_root_model_dir = "test/test_root_model_dir"
+    test_report_to_screen = "Y"
+    git_config = expanduser("~")  + '/.aibench/git'
```

Comment:
Do we want to use the real directory in test?

Reply:
I want to use other directory, but it seems there is no good way for test code to control the generated directory. Is the ~/.aibench/.. configurable from command line?

Reply:
Let's create an argument to specify the .aibench directory later.